### PR TITLE
feat(build script): added build script and initial stark-core module. Closes #13 #45

### LINKED ISSUE #13: Implement the build.sh script
Not having the build.sh script working slows us down too much.

### PR DESCRIPTION
**Pull request by dsebastien**
Added initial build script.
Some improvements can be done over time: https://github.com/dsebastien/stark/blob/ff9444dad1592a8ff249b950e681c4d9f6a81e19/build.sh

We expect some issues with tsc, tsconfig.json files, ... :)
Also the build probably fails with the TSC of packages

Still missing:
* update of CONTRIBUTING.md with new correct commands

### CODE REVIEW

File: build-functions.sh [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+  local PACKAGE_JSON_FILE="$3"
+  
+  local TGZ_PATH="file:..\/dist\/packages-dist\/$PACKAGE\/nationalbankbelgium-$PACKAGE-$VERSION.tgz"
+  logTrace "TGZ path: $TGZ_PATH"
+  
+  local NEWVALUE="\\\"\@nationalbankbelgium\/$PACKAGE\\\": \\\"$TGZ_PATH\\\""
+  
+  perl -p -i -e "s/\"\@nationalbankbelgium\/$PACKAGE\"\s*\:\s*\".*\"/$NEWVALUE/g" $PACKAGE_JSON_FILE 2> /dev/null
+  
+  #perl -p -i -e "s/\"\@nationalbankbelgium\/$PACKAGE\"\s*\:\s*\".*\"/$NEWVALUE/g" ./starter/package.json
```
Comment (SuperITMan) [inline build-functions.sh]: This line can be removed

File: setup.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"use strict";
```
Comment (SuperITMan) [inline setup.js]: What is the goal of this file ? Could we remove it ?

File: build-functions.sh (hunk 1 above)
Comment (dsebastien) [reply build-functions.sh]: Indeed

File: setup.js (hunk 2 above)
Comment (dsebastien) [reply setup.js]: Initial stab at the build we abandoned in favor of perl :). I'll remove it

**Comment by dsebastien** [comment]
I've fixed the build (missing lines) and updated CONTRIBUTING.md

**Pull request merged**